nixos/modules/services/torrent/mldonkey.nix

```diff
@@ -0,0 +1,63 @@
+ # Module to run mldonkey as a daemon
+ 
+ { config, lib, pkgs, ... }:
+ 
+ with lib;
+ 
+ let
+  cfg = config.services.mldonkey;
+ in
+ {
+    ###### interface
+    
+    options = {
+      services.mldonkey = {
+      
+        enable = mkEnableOption "mldonkey daemon";
+               
+        allowedIps = mkOption {
+          type = types.str;
```

Comment:
I believe this makes more sense as `types.listOf types.str`, if the intention is to support multiple IP addresses.